micro-rdk/src/common/conn/viam.rs

```diff
@@ -51,7 +51,7 @@ use super::server::{IncomingConnectionManager, WebRtcConfiguration};
 use crate::common::provisioning::server::AsNetwork;
 
 #[cfg(feature = "ota")]
-use crate::common::{credentials_storage::OtaMetadataStorage, ota};
```

Comment:
I've left this here for now instead of shifting it to `config_monitor.rs`. I like that it's part of `ViamServerStorage`, especially if we remove the flag entirely to make it a builtin